packages/frontend-api/apiary.apib

```diff
@@ -1699,3 +1699,112 @@ Returns validation error
             }
         ]
     }
+
+## Current customer user personal data [/graphql{?current_customer_user_personal_data}]
+
+### POST [POST]
+
+Get current customer user personal data
+
+- Request (application/json)
+    - Headers
+        Authorization: Bearer --ACCESS-TOKEN--
+
+    - Body
+        {
+            query: currentCustomerUser {
+                firstName,
+                lastName,
+                email
+                telephone
+            }
+        }
+
+- Response 200 (application/json; charset=utf-8)
+
+    {
+        "data": {
+            "query": {
+                "firstName": "Jaromír",
+                "lastName": "Jágr",
+                "email": "no-reply@shopsys.com",
+                "telephone": "605000123"
```

Comment:
I hope, it's not  for real, otherwise it could be GDPR issue. :)

Reply:
I think, it is not, feel free to try it :)